Plugin.php

```diff
@@ -32,4 +43,63 @@ public function activate(Composer $composer, IOInterface $io)
             file_put_contents($file, "<?php\n\nreturn [];\n");
         }
     }
+
+    /**
+     * @inheritdoc
+     * @return array The event names to listen to.
+     */
+    public static function getSubscribedEvents()
+    {
+        return [
+            PackageEvents::POST_PACKAGE_UPDATE => 'checkPackageUpdates',
+            ScriptEvents::POST_UPDATE_CMD => 'showUpgradeNotes',
+        ];
+    }
+
+
+    /**
+     * Listen to POST_PACKAGE_UPDATE event and take note of the package updates.
+     * @param PackageEvent $event
+     */
+    public function checkPackageUpdates(PackageEvent $event)
+    {
+        $operation = $event->getOperation();
+        if ($operation instanceof UpdateOperation) {
+            $this->_packageUpdates[$operation->getInitialPackage()->getName()] = [
+                'from' => $operation->getInitialPackage()->getVersion(),
+                'fromPretty' => $operation->getInitialPackage()->getPrettyVersion(),
+                'to' => $operation->getTargetPackage()->getVersion(),
+                'toPretty' => $operation->getTargetPackage()->getPrettyVersion(),
+                'direction' => $event->getPolicy()->versionCompare(
+                    $operation->getInitialPackage(),
+                    $operation->getTargetPackage(),
+                    '<'
+                ) ? 'up' : 'down',
+            ];
+        }
+    }
+
+    /**
+     * Listen to POST_UPDATE_CMD event to display information about upgrade notes if appropriate.
+     * @param Script\Event $event
+     */
+    public function showUpgradeNotes(Script\Event $event)
+    {
+        if (isset($this->_packageUpdates['yiisoft/yii2'])) {
+
+            $package = $this->_packageUpdates['yiisoft/yii2'];
+
+            $io = $event->getIO();
+
+            $io->write("\n  Seems you have "
```

Comment:
wrapping it in `<warning>` looks really ugly:

![bildschirmfoto von 2016-12-15 03 14 25](https://cloud.githubusercontent.com/assets/189796/21209462/af139510-c274-11e6-817f-c12b0359a629.png)

I have come up with this:

![bildschirmfoto von 2016-12-15 03 15 25](https://cloud.githubusercontent.com/assets/189796/21209475/c1638bb2-c274-11e6-9061-8ca713d41efa.png)